src/net.cpp

```diff
@@ -2435,16 +2474,33 @@ void CConnman::ThreadOpenConnections(const std::vector<std::string> connect)
     bool add_fixed_seeds = gArgs.GetBoolArg("-fixedseeds", DEFAULT_FIXEDSEEDS);
     const bool use_seednodes{gArgs.IsArgSet("-seednode")};
 
+    auto seed_node_timer = NodeClock::now();
```

Comment:
It's a bit messy that we are mixing `GetTime` and `NodeClock::now()` overlapping with each other. I made an attempt at making them compatible in 934449f439f5a869177f5865436f1114da5da6bc but needs more consideration.

Reply:
I agree, but I don't think this is something that should be addressed in this PR. This is the case in other places of the codebase, a PR cleaning this up would be appreciated IMO

Reply:
@maflcko Had not seen that, good that things are happening in this area.

@sr-gi Given current master branch, I agree with you in deferring this kind of change.